client/src/resources/lang/en_us.json

```diff
@@ -1180,6 +1188,20 @@
     "wiki.article.standard.noDeathCause.text":"No Death Cause is a game modifier that changes the game such that every grave does not have a death cause listed.",
     "wiki.article.standard.roleSetGraveKillers.title":"Role Set Grave Killers",
     "wiki.article.standard.roleSetGraveKillers.text":"Role Set Grave Killers is a game modifier that changes the game such that every grave killer which is a role is replaced with one of:\n* Syndicate;\n* Town;\n* Minions;\n* Neutral;\n* Fiends; or\n* Cult.\n\nOr, if the killer's role is not in one of these role sets, it is not changed.",
+    "wiki.article.standard.noDueProcess.title": "No Due Process",
+    "wiki.article.standard.noDueProcess.text": "No Due Process is a game modifier that changes the game such that every nominated player will be executed without judgement.",
+    "wiki.article.standard.twoThirdsMajority.title": "Two-thirds Majority",
+    "wiki.article.standard.twoThirdsMajority.text": "Two-thirds Majority is a game modifier that changes the game such that during nomination and judgement, a two-thirds majority is required, rather than a simple majority.",

```

Comment:
two thirds rounded down or up?

Reply:
The smallest integer greater than or equal to two thirds

Reply:
So yeah rounded up

Reply:
Please make that clear in the encyclopedia